GitUI/LeftPanel/RepoObjectsTree.cs

```diff
@@ -608,7 +608,7 @@ private IEnumerable<NodeBase> GetSelectedNodes()
 
         private void OnNodeClick(object sender, TreeNodeMouseClickEventArgs e)
         {
-            NodeBase node = e.Node.Tag as NodeBase;
+            NodeBase node = (NodeBase)e.Node.Tag;
```

Comment:
Why is this better? "As" won't throw, this will.

Reply:
When looking at the usage of the `node` variable it reveals that the logic that follows does not make sense with `node` being `null` and that you would get a nullref exception meaning it is not valid to proceed further with null.
In this particular case using `as` "lies" that `null` is a valid case and something reasonable would be done later.